2023-03-29 14:18:16,264 INFO    MainThread:1339 [wandb_setup.py:_flush():76] Configure stats pid to 1339
2023-03-29 14:18:16,265 INFO    MainThread:1339 [wandb_setup.py:_flush():76] Loading settings from /home/seuyon0101/.config/wandb/settings
2023-03-29 14:18:16,265 INFO    MainThread:1339 [wandb_setup.py:_flush():76] Loading settings from /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/settings
2023-03-29 14:18:16,265 INFO    MainThread:1339 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-29 14:18:16,265 INFO    MainThread:1339 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-29 14:18:16,265 INFO    MainThread:1339 [wandb_init.py:_log_setup():506] Logging user logs to /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/run-20230329_141816-z5b3vagh/logs/debug.log
2023-03-29 14:18:16,265 INFO    MainThread:1339 [wandb_init.py:_log_setup():507] Logging internal logs to /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/run-20230329_141816-z5b3vagh/logs/debug-internal.log
2023-03-29 14:18:16,265 INFO    MainThread:1339 [wandb_init.py:_jupyter_setup():452] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f84c7eb12b0>
2023-03-29 14:18:16,265 INFO    MainThread:1339 [wandb_init.py:init():546] calling init triggers
2023-03-29 14:18:16,265 INFO    MainThread:1339 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {'model_name': 'Vanilla_Transformer', 'n_layers': 6, 'd_model': 512, 'n_heads': 8, 'd_ff': 2048, 'src_vocab_size': 16009, 'tgt_vocab_size': 16009, 'pos_len': 512, 'dropout': 0.2, 'shared': True, 'warmups': 8500, 'epochs': 1, 'optimizer': 'ADAM', 'loss': 'SparseCategoricalCrossentropy', 'metric': 'bleu'}
2023-03-29 14:18:16,265 INFO    MainThread:1339 [wandb_init.py:init():568] re-initializing run, found existing run on stack: 0ztlwaz0
2023-03-29 14:18:16,267 INFO    MainThread:1339 [wandb_run.py:_finish():1854] finishing run saturi/model_exp_spm_16000_0329_final_최종/0ztlwaz0
2023-03-29 14:18:16,267 INFO    MainThread:1339 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-29 14:18:16,267 INFO    MainThread:1339 [wandb_run.py:_restore():2066] restore
2023-03-29 14:18:16,268 INFO    MainThread:1339 [wandb_run.py:_restore():2072] restore done
2023-03-29 14:18:19,799 INFO    MainThread:1339 [wandb_run.py:_footer_history_summary_info():3422] rendering history
2023-03-29 14:18:19,800 INFO    MainThread:1339 [wandb_run.py:_footer_history_summary_info():3454] rendering summary
2023-03-29 14:18:19,803 INFO    MainThread:1339 [wandb_run.py:_footer_sync_info():3380] logging synced files
2023-03-29 14:18:19,807 INFO    MainThread:1339 [wandb_init.py:init():602] starting backend
2023-03-29 14:18:19,807 INFO    MainThread:1339 [wandb_init.py:init():606] setting up manager
2023-03-29 14:18:19,810 INFO    MainThread:1339 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-03-29 14:18:19,813 INFO    MainThread:1339 [wandb_init.py:init():613] backend started and connected
2023-03-29 14:18:19,823 INFO    MainThread:1339 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2023-03-29 14:18:19,823 INFO    MainThread:1339 [wandb_run.py:_label_probe_notebook():1214] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-29 14:18:19,823 INFO    MainThread:1339 [wandb_init.py:init():701] updated telemetry
2023-03-29 14:18:19,937 INFO    MainThread:1339 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-03-29 14:18:20,954 INFO    MainThread:1339 [wandb_run.py:_on_init():2133] communicating current version
2023-03-29 14:18:21,024 INFO    MainThread:1339 [wandb_run.py:_on_init():2142] got version response 
2023-03-29 14:18:21,024 INFO    MainThread:1339 [wandb_init.py:init():789] starting run threads in backend
2023-03-29 14:18:24,963 INFO    MainThread:1339 [wandb_run.py:_console_start():2114] atexit reg
2023-03-29 14:18:24,963 INFO    MainThread:1339 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-29 14:18:24,963 INFO    MainThread:1339 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-29 14:18:24,963 INFO    MainThread:1339 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-29 14:18:24,964 INFO    MainThread:1339 [wandb_init.py:init():831] run started, returning control to user process
2023-03-29 14:42:04,381 INFO    MainThread:1339 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-03-29 14:42:04,381 INFO    MainThread:1339 [wandb_init.py:_pause_backend():417] pausing backend
2023-03-29 14:56:07,380 INFO    MainThread:1339 [wandb_init.py:_resume_backend():422] resuming backend
2023-03-29 14:56:07,395 INFO    MainThread:1339 [wandb_setup.py:_flush():76] Configure stats pid to 1339
2023-03-29 14:56:07,396 INFO    MainThread:1339 [wandb_setup.py:_flush():76] Loading settings from /home/seuyon0101/.config/wandb/settings
2023-03-29 14:56:07,396 INFO    MainThread:1339 [wandb_setup.py:_flush():76] Loading settings from /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/settings
2023-03-29 14:56:07,396 INFO    MainThread:1339 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-29 14:56:07,396 INFO    MainThread:1339 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-29 14:56:07,396 INFO    MainThread:1339 [wandb_init.py:_log_setup():506] Logging user logs to /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/run-20230329_145607-2lne7mkb/logs/debug.log
2023-03-29 14:56:07,396 INFO    MainThread:1339 [wandb_init.py:_log_setup():507] Logging internal logs to /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/run-20230329_145607-2lne7mkb/logs/debug-internal.log
2023-03-29 14:56:07,396 INFO    MainThread:1339 [wandb_init.py:init():546] calling init triggers
2023-03-29 14:56:07,396 INFO    MainThread:1339 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {'model_name': 'Vanilla_Transformer', 'n_layers': 6, 'd_model': 512, 'n_heads': 8, 'd_ff': 2048, 'src_vocab_size': 16009, 'tgt_vocab_size': 16009, 'pos_len': 512, 'dropout': 0.2, 'shared': True, 'warmups': 8500, 'epochs': 1, 'optimizer': 'ADAM', 'loss': 'SparseCategoricalCrossentropy', 'metric': 'bleu'}
2023-03-29 14:56:07,396 INFO    MainThread:1339 [wandb_init.py:init():568] re-initializing run, found existing run on stack: z5b3vagh
2023-03-29 14:56:07,398 INFO    MainThread:1339 [wandb_run.py:_finish():1854] finishing run saturi/model_exp_spm_16000_0329_final_최종/z5b3vagh
2023-03-29 14:56:07,398 INFO    MainThread:1339 [jupyter.py:save_history():445] not saving jupyter history
2023-03-29 14:56:07,398 INFO    MainThread:1339 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-03-29 14:56:07,399 INFO    MainThread:1339 [wandb_init.py:_jupyter_teardown():434] cleaning up jupyter logic
2023-03-29 14:56:07,399 INFO    MainThread:1339 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-29 14:56:07,399 INFO    MainThread:1339 [wandb_run.py:_restore():2066] restore
2023-03-29 14:56:07,399 INFO    MainThread:1339 [wandb_run.py:_restore():2072] restore done
2023-03-29 14:56:10,509 INFO    MainThread:1339 [wandb_run.py:_footer_history_summary_info():3422] rendering history
2023-03-29 14:56:10,509 INFO    MainThread:1339 [wandb_run.py:_footer_history_summary_info():3454] rendering summary
2023-03-29 14:56:10,513 INFO    MainThread:1339 [wandb_run.py:_footer_sync_info():3380] logging synced files
